index.js

```diff
@@ -74,6 +74,10 @@ function RedisCache(options) {
     function() {
       return DEFAULT_RETRY_MS;
     };
+  this.maxAge = Number(options.maxAge)
+  if (options.maxAge && !this.maxAge) {
```

Comment:
Lite otydlig kod, används this.maxAge sen? Kanske:
```
if (options.maxAge && isNaN(options.maxAge)) ...
```

Reply:
make it throw

Reply:
kolla specialfall för "". Number("") ger 0. Och tomma strängen antyder oändlig cache tid, 0 ingen. Det blir konstigt semantiskt.

Reply:
Nu beter den sig samma om man skickar maxAge: "" eller ingen maxAge alls.